pyproject.toml

```diff
@@ -433,7 +433,7 @@ max-bool-expr = 5
 max-branches = 20
 
 # Maximum number of locals for function / method body.
-max-locals = 19
+max-locals = 20
```

Comment:
not allowed to change, we need to look at why we have so many functions calls.

Reply:
done.